google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/flexible/AppEngineFlexibleSupportProvider.java

```diff
@@ -112,7 +114,20 @@ public static void addSupport(@NotNull AppEngineFlexibleFacet facet,
       facet.getConfiguration().setDockerDirectory(dockerDirectory.toString());
 
       if (generateConfigFiles) {
-        appEngineProjectService.generateAppYaml(FlexibleRuntime.JAVA, facet.getModule(), appYamlPath.getParent());
+        UsageTrackerProvider.getInstance()
```

Comment:
should this method also track adding of support? (aside from just adding the config)

See AppEngineStandardSupportProvider#addSupport